File: src/com/ecci/Hamers/Quote.java
package com.ecci.Hamers;

public class Quote {

    private String user;
    private String body;
    private String date;

    public Quote(String user, String body, String date) {
        super();
        this.user = user;
        this.body = body;
        this.date = date;
    }

    public String getBody() {
        return body;
    }

    public String getDate() {
        return date;
    }

    public String getUser() {
        return user;
    }
}

File: hamersapp/src/main/java/nl/ecci/hamers/models/Quote.kt
package nl.ecci.hamers.models

import com.google.gson.annotations.SerializedName
import java.util.*

internal class Quote(val id: Int,
                     val text: String,
                     @SerializedName("user_id")
                     val userID: Int,
                     @SerializedName("created_at")
                     val date: Date)